src/lib.rs

```diff
@@ -850,7 +861,13 @@ mod tests {
 
         #[test]
         fn capture_stderr() {
-            let Stderr(stderr) = run_output!(test_helper(), "write to stderr");
+            let script = TestScript::new(
+                r#"
+                    import sys
+                    print("foo", file=sys.stderr)
+                "#,
```

Comment:
If you use single quotes you can avoid raw strings:

```suggestion
                "
                    import sys
                    print('foo', file=sys.stderr)
                ",
```